plugin-name/webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
+const path = require( 'path' );
+
+const entry = {
```

Comment:
Maybe that code can improved with a loop?

Reply:
good idea! updated the webpack config in commit 3bcc2cd1